srcpkgs/python3-mypy/template

```diff
@@ -1,17 +1,20 @@
 # Template file for 'python3-mypy'
 pkgname=python3-mypy
-version=0.790
-revision=2
+version=0.812
+revision=1
 wrksrc="mypy-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools python3-Sphinx"
 depends="python3-mypy_extensions python3-typed-ast python3-typing_extensions"
+checkdepends="$depends python3-pytest python3-pip python3-lxml python3-devel
+ python3-virtualenv"
```

Comment:
Presence of pip and -devel make me think it's installing a bunch of packages from pypi. I wonder how we could make sure it isn't, assuming they are actually necessary for tests...

If the tests do rely on `pip`, I'm not sure we should be running them.